maps/unused/mushroom_new_walls.dmm

```diff
@@ -17586,6 +17639,17 @@
 /obj/mapping_helper/access/mining,
 /turf/simulated/floor/plating/airless,
 /area/station/mining/magnet)
+"eyb" = (
+/obj/cable{
+	icon_state = "0-2"
+	},
+/obj/cable,
+/obj/mapping_helper/wingrille_spawn/auto/reinforced/crystal,
+/obj/machinery/turretid/armory{
+	pixel_x = -24
+	},
```

Comment:
How am I going to reach my turret controls if they are outside and encased in glass

Reply:
accidental second misplace, still in original location just with the new preset